Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/InferableSingleKey.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Specialized;
+using KindMen.Uxios;
+using Netherlands3D.Web;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class InferableSingleKey : StoredAuthorization
+    {
+        public string key = "";
```

Comment:
Voor de duidelijkheid misschien beter queryKeyValue noemen? Dat is consistent met hieronder, en dit gaat altijd over query strings (tenminste, als mijn feedback bij bearertoken opgelost is)